packages/protocol/scripts/create-stable-token.ts

```diff
@@ -0,0 +1,265 @@
+import * as fs from 'fs'
+
+const KEY_TO_BE_REPLACED_IN_TEMPLATES = '{TICKER}'
+
+const StableTokenTemplate = `pragma solidity ^0.5.13;
+
+import "./StableToken.sol";
+
+contract StableToken{TICKER} is StableToken {
+  \/**
+   * @notice Sets initialized == true on implementation contracts.
+   * @param test Set to true to skip implementation initialization.
+   */
+  constructor(bool test) public StableToken(test) {}
+
+  /**
+   * @notice Returns the storage, major, minor, and patch version of the contract.
+   * @dev This function is overloaded to maintain a distinct version from StableToken.sol.
+   * @return The storage, major, minor, and patch version of the contract.
+   */
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
+    return (1, 1, 0, 0);
+  }
+}
+`
+
+const ExchangeTemplate = `pragma solidity ^0.5.13;
+
+import "./Exchange.sol";
+
+contract Exchange{TICKER} is Exchange {
+  \/**
+   * @notice Sets initialized == true on implementation contracts
+   * @param test Set to true to skip implementation initialization
+   */
+  constructor(bool test) public Exchange(test) {}
+
+  /**
+   * @notice Returns the storage, major, minor, and patch version of the contract.
+   * @dev This function is overloaded to maintain a distinct version from Exchange.sol.
+   * @return The storage, major, minor, and patch version of the contract.
+   */
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
+    return (1, 1, 0, 0);
+  }
+}
+`
+
+const StableTokenProxyTemplate = `pragma solidity ^0.5.13;
+
+import "../../common/Proxy.sol";
+
+/* solhint-disable no-empty-blocks */
+contract StableToken{TICKER}Proxy is Proxy {}
+`
+
+const ExchangeProxyTemplate = `pragma solidity ^0.5.13;
+
+import "../../common/Proxy.sol";
+
+/* solhint-disable no-empty-blocks */
+contract Exchange{TICKER}Proxy is Proxy {}
+`
+
+const migrationStableTemplate = `/* tslint:disable:no-console */
+import { ensureLeading0x, eqAddress, NULL_ADDRESS } from '@celo/base/lib/address'
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import {
+  deploymentForCoreContract,
+  getDeployedProxiedContract,
+} from '@celo/protocol/lib/web3-utils'
+import { config } from '@celo/protocol/migrationsConfig'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import {
+  FeeCurrencyWhitelistInstance,
+  FreezerInstance,
+  ReserveInstance,
+  SortedOraclesInstance,
+  StableToken[KEY_TO_BE_REPLACED_IN_TEMPLATES}Instance,
```

Comment:
Should all instances of `KEY_TO_BE_REPLACED_IN_TEMPLATES` be `{TICKER}` instead? The former is the name of a variable, the latter is the actual value of it.

Reply:
Also looks like a typo here, in that a closing `[` and an opening `}` is missing.